contracts/VoltzPausabilityWrapper.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity =0.8.9;
+
+import "./VAMM.sol";
```

Comment:
why not just use the IVAMM interface instead of VAMM.sol as a dependency?

Reply:
paused() is in Pausable contract that is inherited by the VAMM. IVAMM doesn't know about it.